apps/aechannel/src/aesc_fsm.erl

```diff
@@ -98,7 +98,7 @@
                   | ?SHUTDOWN
                   | ?SHUTDOWN_ACK.
 
--define(DUMMY_BLOCK_HASH, <<12345:32/unit:8>>).
+-define(NOT_SET_BLOCK_HASH, <<0:32/unit:8>>).
```

Comment:
I see this value used for matching.

Is this field persisted from requests produced by node `v2.*`?

Reply:
@velzevur Thanks for pointing out this was not released in stable release yet. 🤦‍♂ So good.